examples/reproduce_testcases.jl

```diff
@@ -49,31 +49,40 @@ function test_diamond_heisenberg_sf()
     # This is implicitly in the same units as the units you use to define
     #  the interactions in the Hamiltonian. Here, we defined our interactions
     #  in K, but we want to see ω in units of meV (to compare to a baseline
-    #  solution we have).
-    # We additionally need to scale by (S+1) with S=3/2 to match our reference,
-    #  which is a spin-3/2 model.
-    maxω = 2π / (meas_rate * Δt) * (1000 * kB) / (5/2)
-    p = plot_many_cuts(avgS; maxω=maxω, chopω=5.0)
+    #  linear spin-wave solution we have).
+    maxω = 2π / (meas_rate * Δt) * (1000 * kB)
+    p = plot_many_cuts_afmdiamond(avgS, 1000 * kB * J, 3/2; maxω=maxω, chopω=5.0)
+
     display(p)
     return avgS
 end
 
+# Result for the NN AFM diamond from linear spin-wave theory
+# `q` should be in fractional coordinates in the conventional recip lattice
+function linear_sw_diamond_heisenberg(q, J, S)
+    exp1 = exp(im * π * (q[1] + q[2]))
+    exp2 = exp(im * π * (q[2] + q[3]))
+    exp3 = exp(im * π * (q[1] + q[3]))
+    return 4 * J * S * √(1 - abs(1 + exp1 + exp2 + exp3)^2 / 16)
+end
+
 """
-    plot_S_cut(S, qz; max_ω, chop_ω)
+    plot_S_cut_afmdiamond(S, qz; max_ω, chop_ω)
 
 Plots cuts through the structure factor:
     (0, 0, qz) -> (π, 0, qz) -> (π, π, qz) -> (0, 0, 0)
-where qz is (c⃰ * qz). Only works on cubic lattices. To get unitful
+where qz is (c⃰ * qz). Assumes that S has 1BZ along (a, b) and
```

Comment:
Just now noticing the notation `c⃰`. Is this standard (asterisk directly above c?) If not, maybe overbar?

Reply:
The standard is an asterisk superscript, not directly over the c. I think I should just not try to use the unicode "asterisk above", and use "^*".

Reply:
Sounds good. To be clear, the meaning is that `c^*` is the reciprocal lattice vector corresponding to lattice vector `c`?

Reply:
Yup!